.github/workflows/sycl-linux-run-tests.yml

```diff
@@ -281,6 +281,7 @@ jobs:
     - name: Run E2E Tests
       if: inputs.tests_selector == 'e2e'
       uses: ./devops/actions/run-tests/e2e
+      timeout-minutes: 65
```

Comment:
```suggestion
      timeout-minutes: 60
```

Should be fine...

Reply:
i did 65 because now the timeout includes checkout/pack/upload

Reply:
I understand, but 60 was too generous to begin with. 55 should be enough.

Reply:
ok will change